src/tests/integration/mapping.test.ts

```diff
@@ -42,4 +40,36 @@ describe('/api', () =>{
       expect(data.fullDomain).toEqual(`${subDomain}.${domain}`)
     })
   })
+
+  it('Delete mapping', async()=>{
+    const subDomain = `delete${uuidv4()}`
+    const domain = 'albertow'
+    const port = '4500'
+    await fetch(`${apiURL}/api/mappings`, {
```

Comment:
let's create a helper function, there's alot of duplicate code.
```
const mappingAdapter = (path='/', method, body) => {
  const options = {
    method, headers: {
    ....
    }
  }
  if (body) {
    options.body = JSON.stringify(body)
  }
  return fetch(`${apiURL}/api/mappings${path}`, options)
}
```

Then to use this:
```
await mappingAdapter('/', 'POST', {domain, subDomain, port})
const deleteResponse = await mappingAdapter(`/${data.id}`, 'POST', {domain, subDomain, port})
const deleteResult = await deleteResponse.json()
expect(deleteResult.port).toEqual(port)
.....
```

Reply:
Should create a refactor story.

Reply:
you or @rkalra247 can pick this one up.

Reply:
Can be marked as resolved for now.

Reply:
I'm already done, just need to push up.